02-peer-protocol.md

```diff
@@ -1445,10 +1445,10 @@ A node:
     - if `next_commitment_number` is not 1 greater than the
```

Comment:
I started with the smallest changes possible, but I find it weird that we have those requirements in the `A node` section and not in the `A receiving node` section, they only make sense once you've received the remote `channel_reestablish`, don't they? Should I move those in the `receiving node` section below?

Reply:
Yeah this also tripped me up as well when re-reading this section recently (had been a looong time since I had to dive into this section). I recall that during the past spec meeting in Oakland, @rustyrussell had a sort of mini session near the end explaining how to properly write out these types of requirements to minimize ambiguity....don't think any one took notes though